### ISSUE TITLE
Fine tune

### ISSUE THREAD

**Issue by BIT-MJY** [qiaozhijian/LPD-Net-Pytorch#2]
Hello @qiaozhijian,

Thanks for sharing your reproduced PyTorch-version LPD-Net.

Now I'm trying to train the PyTorch-version LPD with sequence 2012-01-08 of the NCLT dataset, only using 3D points without local features. However, the evaluation on  2012-02-05 (around a 1-month time gap) shows that the recall@1 of LPD can only achieve 50%~60%, while our own method with only input range images can achieve 85%~90%. I think maybe the setup of the PyTorch-version LPD I'm using needs to change.

Thus could you please tell me that 
1) what is the performance of your PyTorch-version LPD on two sequences with a 1-month time gap of NCLT dataset (or Robotcar dataset)?
2) how does only using point clouds without extracted local features influence the recall on the NCLT dataset (or Robotcar dataset)?
3) Is there any default setup that needs to be changed for better fine-tuning results in your PyTorch-version LPD?

Thank you very much!

**Comment by qiaozhijian** [comment]
1. well, I have provided my pretrained model, and you can try it directly. You may need to process data from NCLT like that from Robotcar dataset.
2. You can refer to the paper. But I think there is no need to compute local features for only littile improvement. Local features extraction is time-consuming.
3. There are some common tricks for finetune. Small learning rate, only fine tune some layers and etc. The most important point: point cloud is zoomed, ground-removed and scaled when trained in RobotCar dataset. You must keep them consistent.

**Comment by BIT-MJY** [comment]
Thanks for your suggestion. I will try them.
